sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample9_DifferentiateOutputModelsTrainedWithAndWithoutLabels.cs

```diff
@@ -0,0 +1,118 @@
+
+            using (FileStream stream = new FileStream(formFilePath, FileMode.Open))
+            {
+                RecognizedFormCollection formsWithLabels = await client.StartRecognizeCustomForms(modelWithLabels.ModelId, stream).WaitForCompletionAsync();
+
+                // With a form recognized by a model trained with labels, the 'field.Name' key will be its
+                // training-time label. Note that Label data is not returned for model trained with labels.
+                Console.WriteLine("---------Recognizing forms with models trained with labels---------");
+                foreach (RecognizedForm form in formsWithLabels)
+                {
+                    Console.WriteLine($"Form of type: {form.FormType}");
+                    foreach (FormField field in form.Fields.Values)
+                    {
+                        Console.WriteLine($"Field {field.Name}: ");
+                        Console.WriteLine($"    Value: '{field.ValueText.Text}");
+                        Console.WriteLine($"    Confidence: '{field.Confidence}");
+                    }
+                }
+
+                // Find a specific labeled field.
+                // For this particular sample, we will look for the known training-time label 'VendorName'.
+                Console.WriteLine("Find the value for a specific labeled field:");
+                foreach (RecognizedForm form in formsWithLabels)
+                {
+                    if (form.Fields.TryGetValue("VendorName", out FormField field))
+                    {
+                        Console.WriteLine($"VendorName is {field.ValueText.Text}");
+                    }
+                }
+            }
+        }
+
+        [Test]
+        public async Task OutputModelsTrainedWithoutLabels()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            string trainingFileUrl = TestEnvironment.BlobContainerSasUrl;
+            string formFilePath = FormRecognizerTestEnvironment.CreatePath("Form_1.jpg");
+
+            FormTrainingClient trainingClient = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Model trained without labels
+            CustomFormModel modelWithoutLabels = await trainingClient.StartTraining(new Uri(trainingFileUrl), useTrainingLabels: false).WaitForCompletionAsync();
+
+            using (FileStream stream = new FileStream(formFilePath, FileMode.Open))
+            {
+                RecognizedFormCollection formsWithoutLabels = await client.StartRecognizeCustomForms(modelWithoutLabels.ModelId, stream).WaitForCompletionAsync();
+
+                // With a form recognized by a model trained without labels, the 'field.Name' property will be denoted by a numeric index.
```

Comment:
Can we also call out that non-unique Form Field Labels will be included in the field.LabelText ... this is a chance to show them that while the unique names aren't super useful, they can find what they'll think of as the field name here.

Reply:
I'm not sure I follow.... could you explain more what you mean?

Reply:
See [here](https://github.com/Azure/azure-sdk-for-net/blob/27003d49f3b689f41b38371b329dde00627513cc/sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs#L81) where we create the unique dictionary key.

I think users will be more interested in the non-unique `fieldLabel` value, which will be something like "Vendor Name" rather than "field-0".  We should show them that this value is there.

Reply:
Ohhh I get your comment now. Yes. This a great point. I will add

Reply:
updated